app/css/style.css

```diff
@@ -1,3 +1,336 @@
+/*
```

Comment:
I should just include this in the packages.json file and not include it any further in the projects. Just let it build in the gulp task.